ambry-router/src/main/java/com.github.ambry.router/PutOperation.java

```diff
@@ -518,13 +518,20 @@ Exception getOperationException() {
 
   /**
    * The time at which this operation was submitted.
-   * @return return the time at which the operation was submitted.
+   * @return the time at which the operation was submitted.
    */
   long getSubmissionTimeMs() {
     return submissionTimeMs;
   }
 
   /**
+   * @return if this is a composite object, list of successfully put chunk ids; null otherwise.
+   */
+  ArrayList<String> getSuccessfullyPutChunkIds() {
```

Comment:
Is it a better way to reuse a global `ArrayList` so there is no need to new a list every time?